cpp/src/gandiva/expression_registry.h

```diff
@@ -37,7 +38,8 @@ class GANDIVA_EXPORT ExpressionRegistry {
  public:
   using native_func_iterator_type = const NativeFunction*;
   using func_sig_iterator_type = const FunctionSignature*;
-  ExpressionRegistry();
+  ExpressionRegistry(
+      FunctionRegistry* function_registry = gandiva::default_function_registry());
```

Comment:
Perhaps make this explicit:
```suggestion
  explicit ExpressionRegistry(
      FunctionRegistry* function_registry = gandiva::default_function_registry());
```

Reply:
Fixed.